react/src/pages/AccountsComponent.js

```diff
@@ -2,14 +2,34 @@ import React, { useEffect, useState } from 'react';
 import { BarSpinner, StandardSpinner } from '../Spinners';
 import { useSearchParams } from 'react-router-dom';
 import Char from '../utils/Char';
+import Client from '../utils/Client';
```

Comment:
These changes are unrelated to this fix - just a minor UI update to the accounts page to include a link to the SSL certificate info for Portal and Foursight, and to include the portal access key ID for the current account.